tests/unit/chroma_core/helpers/helper.py

```diff
@@ -61,11 +61,13 @@ def generate_csr(common_name):
         stdout=subprocess.PIPE,
         stderr=subprocess.PIPE,
     )
+
```

Comment:
nit: are these empty lines necessary? Might be able to just revert this file.

Reply:
Or maybe `black` updated the fie?

Reply:
I added them. I agree they don't serve any real purpose besides statement separation, but don't see any harm in them being here.

Reply:
for sure 👍